transmart-batch/docs/data_formats/clinical_data_file.md

```diff
@@ -47,8 +47,20 @@ Observations derived from electronic health records can be uploaded as part of a
 | patient2   | 1             |        |               | 20.9 | 82         |          |
 | patient2   | 2             |        |               |      | 69         | 142      |
 
+### Sample-specific data (not yet implemented)
+In case some of your clinical data is specific to a subject's sample (used in a high-dimensional data set), the   
+`Sample CD` modifier can be used. When applied, all observations in that row will be linked to the sample.
```

Comment:
I didn't know this was possible. Is this your own idea or are you sure that this will not interfere with High Dim data retrieval? Can it be used to select all (low and high dim) observations linked to a Sample ID?